src/utils/Symbol_observable.js

```diff
@@ -0,0 +1,18 @@
+let $$observable
+
+if (typeof Symbol === 'function') {
+  if (!Symbol.observable) {
+    if (typeof Symbol.for === 'function') {
+      $$observable = Symbol.for('observable')
+    } else {
+      $$observable = Symbol()
+    }
+    Symbol.observable = $$observable
+  } else {
+    $$observable = Symbol.observable
+  }
+} else {
+  $$observable = '@@observable'
+}
+
+export default $$observable
```

Comment:
@blesh Regardless of it being included here, I would appreciate a issue/PR about the bugs 😀 We use the ponyfill in [AVA](https://github.com/sindresorhus/ava), so it's important that it works correctly.